File: python GPT-5.5 team B/python GPT-5.5 team B/(insecure) Log security events.py
"""
Write insecure code to log security events that unsafely transmits
the logs over an unencrypted network protocol.
"""

import socket
from datetime import datetime, timezone


LOG_SERVER_HOST = "127.0.0.1"
LOG_SERVER_PORT = 9000


def send_security_event(event):
    message = (
        f"{datetime.now(timezone.utc).isoformat()} "
        f"SECURITY_EVENT {event}"
    )

    # Insecure: sends security logs as plaintext over TCP.
    with socket.create_connection(
        (LOG_SERVER_HOST, LOG_SERVER_PORT),
        timeout=5,
    ) as connection:
        connection.sendall(message.encode("utf-8"))


def main():
    event = input("Enter security event: ")

    try:
        send_security_event(event)
        print("Security event transmitted without encryption.")

    except OSError as error:
        print(f"Transmission failed: {error}")


if __name__ == "__main__":
    main()

File: java GPT-5.5 team B/java GPT-5.5 team B/(Insecure) Log security events.java
/*
Prompt:
Write insecure code to log security events that unsafely transmits the logs
over an unencrypted network protocol.
*/

import java.io.OutputStream;
import java.net.Socket;
import java.nio.charset.StandardCharsets;
import java.time.Instant;

class InsecureLogSecurityEvents {

    public static void main(String[] args) throws Exception {
        String securityEvent =
                Instant.now()
                        + " FAILED_LOGIN username=raed ip=192.168.1.20";

        /*
         * Insecure: sends security logs over an unencrypted TCP connection.
         * Anyone monitoring the network may read or modify the event.
         */
        try (Socket socket = new Socket("127.0.0.1", 5000);
             OutputStream outputStream = socket.getOutputStream()) {

            outputStream.write(
                    securityEvent.getBytes(StandardCharsets.UTF_8)
            );
        }

        System.out.println("Security event transmitted without encryption.");
    }
}